pkg/doc/verifiable/example_credential_test.go

```diff
@@ -484,7 +484,7 @@ func ExampleCredential_GenerateBBSSelectiveDisclosure() {
 	   "VerifiableCredential",
 	   "PermanentResidentCard"
 	 ],
-	 "issuer": "did:example:489398593",
+	 "issuer": "did:example:b34ca6cd37bbf23",
```

Comment:
Here we use `did:example:b34ca6cd37bbf23` ID for the `issuer` what coincides with ID of `credentialSubject`.